### DISCUSSION TITLE
Using wasmfs opfs without Asyncify=2/JSPI

### DISCUSSION THREAD

**Discussion by ProgrammerIn-wonderland** [emscripten-core/emscripten#22763] [Q&A]
Hey! I'm trying to use wasmfs without using web workers or using JSPI, is this currently possible? If so what emcc command should I use to compile an application with this?

**Comment by sbc100** [comment]
@tlively

**Comment by kripken** [comment]
See

https://github.com/emscripten-core/emscripten/issues/15949#issuecomment-2364658781
